include/ceedf.h

```diff
@@ -154,14 +161,17 @@
 ! Operator and OperatorField Constants
 !-----------------------------------------------------------------------
 
-      integer ceed_basis_collocated
-      parameter(ceed_basis_collocated     = -1)
-
       integer ceed_vector_active
-      parameter(ceed_vector_active        = -1)
+      parameter(ceed_vector_active        = -5)
 
       integer ceed_vector_none
-      parameter(ceed_vector_none          = -2)
+      parameter(ceed_vector_none          = -6)
+
+      integer ceed_elemrestriction_none
+      parameter(ceed_elemrestriction_none = -7)
+
+      integer ceed_basis_collocated
+      parameter(ceed_basis_collocated     = -8)
```

Comment:
I updated these numbers to be unique while I added these fields. It should help make less buggy Fortran code.